### DISCUSSION TITLE
Changing Time Signature

### DISCUSSION THREAD

**Discussion by P-Mouse** [falkTX/Carla#1596] [Q&A]
I have been using Carla VST host now for a while on Windows.

Really loving the interface and all, great kudos for the developers!!

I have only one question about it.
Maybe I am just not looking right, but I just can't seem to find a way to change the time signature?
I play a lot of music in 3/4 signature, as well as 6/8 and an odd song in 7/8.
I can only change the BPM, but there doesn't seem to be an option to change the time signature or am I wrong?

**Comment by falkTX** [comment]
That is correct. I am always in 4/4 so that is what I setup carla for.

**Reply by P-Mouse** [reply]
Will there be a way to change this maybe in the future?
Many other VST hosts are able to do so, so it doesn't seem to be so rare?
Quite a lot of music isn't in 4/4.
